next-cloudinary/.npmignore

```diff
@@ -1,5 +0,0 @@
-babel.config.js
-jest.config.js
-node_modules
-src
```

Comment:
declarations are emitted now in `dist/declarations/src` and this glob caught that so they were npmignored